### ISSUE TITLE
No recreation of a scaleway_server after an image id change

### ISSUE THREAD

**Issue by sieben** [terraform-providers/terraform-provider-scaleway#83]
### Terraform Version

```
$ terraform version
Terraform v0.11.8-dev
```

### Affected Resource(s)

- scaleway_server

### Terraform Configuration Files
```hcl
provider "scaleway" {
  region = "par1"
}

resource "scaleway_server" "example01" {
  name  = "example01"
  image = "27452e61-310e-4fe5-93af-0a0bdf4c20a6"
  type  = "START1-S"
}
```

### Debug Output

https://gist.github.com/sieben/d02669cae255d40f53c6d627acb1c02d

### Expected Behavior

The previous instance should have been destroyed and replace with a new one.

### Actual Behavior

The destroy happened but no creation was performed.

### Steps to Reproduce

1. `terraform apply` with a given image ID
1.  `terraform apply` with a different image ID

**Comment by nicolai86** [comment]
Hey @sieben ,

I’m terribly sorry about this! I’m pretty sure this is related to the change in #72; will investigate ASAP.

**Comment by nicolai86** [comment]
So the change has nothing to do with #72 - but rather #55 . If the server is shut down the task remains in pending state, even after the server is long gone.

Here's an example of the shutdown task: the server is gone, but the task is still pending

```
{
  "task": {
    "status": "pending",
    "description": "server_terminate",
    "terminated_at": null,
    "href_from": "/servers/2c931608-fb84-48ec-a736-68c6d907537e/action",
    "progress": 0,
    "started_at": "2018-08-05T17:13:42.808951+00:00",
    "id": "220f2176-708b-4808-8e50-eaafa3bed373"
  }
}
```

Sounds like #55 should be reverted.

**Comment by nicolai86** [comment]
@sieben does Scaleway provide a recommendation what to use to determine if a server state change succeeded? e.g. is the task api something terraform should be using, or is it recommended to fallback to the previous server resource polling?

**Comment by nicolai86** [comment]
I've reverted #55 for shutdown only as startup/ poweroff seems to be working fine with the task api. Also I've included a new test case for the example @sieben provided - will provide a PR once all tests are green.

**Comment by nicolai86** [comment]
fixed in #87

**Comment by nicolai86** [comment]
this will be fixed with the release of 1.6.0

**Comment by nicolai86** [comment]
1.6.0 was released 🎉
